.github/workflows/BuildPR.yml

```
@@ -0,0 +1,25 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildGitx:
+    name: Build gitx
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        with:
+          fetch-depth: 0
+          submodules: true
+      - name: Set XCode Version
+        run: sudo xcode-select -s /Applications/Xcode_12.2.app
+      - name: Build project
+        run: xcodebuild -workspace ObjectiveGitFramework.xcworkspace -scheme "ObjectiveGit Mac" build | xcpretty
+      - name: Xcodebuild Action
+        uses: sersoft-gmbh/xcodebuild-action@v1.1
+        with:
+          project: ObjectiveGitFramework.xcodeproj
+          scheme: ObjectiveGit Mac
+          configuration: Release
+          action: build
```

```suggestion
          action: archive
```
We did `archive` in the other build. Does it make sense here?